Mage.Sets/src/mage/sets/DominariaUnitedCommander.java

```diff
@@ -78,8 +78,11 @@ private DominariaUnitedCommander() {
         cards.add(new SetCardInfo("Fusion Elemental", 150, Rarity.UNCOMMON, mage.cards.f.FusionElemental.class));
         cards.add(new SetCardInfo("Garna, the Bloodflame", 151, Rarity.UNCOMMON, mage.cards.g.GarnaTheBloodflame.class));
         cards.add(new SetCardInfo("Geier Reach Sanitarium", 213, Rarity.RARE, mage.cards.g.GeierReachSanitarium.class));
-        cards.add(new SetCardInfo("General Marhault Elsdragon", 31, Rarity.UNCOMMON, mage.cards.g.GeneralMarhaultElsdragon.class));
+        cards.add(new SetCardInfo("General Marhault Elsdragon", 31, Rarity.UNCOMMON, mage.cards.g.GeneralMarhaultElsdragon.class, NON_FULL_USE_VARIOUS));
+        cards.add(new SetCardInfo("General Marhault Elsdragon", 53, Rarity.UNCOMMON, mage.cards.g.GeneralMarhaultElsdragon.class, NON_FULL_USE_VARIOUS));
         cards.add(new SetCardInfo("Generous Gift", 100, Rarity.UNCOMMON, mage.cards.g.GenerousGift.class));
+        cards.add(new SetCardInfo("Gerrard's Hourglass Pendant", 17, Rarity.RARE, mage.cards.g.GerrardsHourglassPendant.class, NON_FULL_USE_VARIOUS));
+        cards.add(new SetCardInfo("Gerrard's Hourglass Pendant", 93, Rarity.RARE, mage.cards.g.GerrardsHourglassPendant.class, NON_FULL_USE_VARIOUS));
```

Comment:
you don't have to fix this but for future reference you can just add the one version of the card to the set file and the rest will be added later

Reply:
honestly I'm not sure about that aspect, I think we could actually get rid of `NON_FULL_USE_VARIOUS` but it causes a test failure at the moment